src/components/network/network.js

```diff
@@ -152,6 +152,10 @@ genotet.NetworkView = function(viewName, params) {
     .on('genotet.nodeClick', function(event, node) {
       this.panel.displayNodeInfo(node);
       this.loader.incidentEdges(node);
+      this.signal('link', {
```

Comment:
Because they have some same operations and can share many codes. You can see link-manager.js